frontend/vendor/js/diff.min.js

```diff
@@ -0,0 +1,307 @@
+/*!
+
+ diff v3.1.0
```

Comment:
It would definitely be nice to have this (and all of the other `js` dependencies) in `package.json` instead of [vendor](https://github.com/exercism/exercism.io/tree/master/frontend/vendor/js). I'm not sure how to do that without installing [browserify](http://browserify.org/) or some such `require()`-fu. Is there a simpler way?